### PR TITLE
CONFIGURE: Improve error messages for optional dependencies

### PR DESCRIPTION
**Pull request by lslebodn**
https://www.redhat.com/archives/freeipa-devel/2017-March/msg00307.html

### CODE REVIEW

**Comment by tiran** [comment]
NACK, you are changing the spirit of the accepted PR #502.

**Comment by lslebodn** [comment]
>NACK, you are changing the spirit of the accepted PR #502.

The approach PR #502 was not accepted in upstream discussion
https://www.redhat.com/archives/freeipa-devel/2017-March/msg00307.html

and moreover @HonzaCholasta was not against in PR #502
https://github.com/freeipa/freeipa/pull/502#issuecomment-286724681

File: configure.ac [hunk 1]
```diff
@@ -381,25 +381,19 @@ AC_SUBST([i18ntests])
...
-	    AS_HELP_STRING([--enable-pylint],
-			   [Require pylint. Default is autodetection with
-			    "python -m pylint".]),
+	    AS_HELP_STRING([--disable-pylint],
+			   [skip Pylint in make lint target
+This feature is optional and aimed for checking issues in python code.
+You can skip this check wich configure time option --disable-pylint.]),
 	    [PYLINT=$enableval],
-	    [PYLINT=check]
+	    [PYLINT=yes]
```
Comment (tiran) [inline configure.ac]: NACK. This changes behavior and undos an accepted PR.

**Comment by lslebodn** [comment]
@tiran one more time:
The approach in PR #502 was not accepted in upstream discussion
https://www.redhat.com/archives/freeipa-devel/2017-March/msg00307.html

**Comment by tomaskrizek** [comment]
Closing for inactivity. If this PR is still relevant, please re-open it.

**Pull request closed without merging**